src/mote/modules/call.py

```diff
@@ -44,12 +43,9 @@ def fetch_channel_dict():
 def fetch_datetxt_dict(channel: str):
     try:
         datetxt_dict = {}
-        datelist = os.listdir("/srv/web/meetbot/%s" % channel)
+        datelist = f"{os.listdir(app.config['MEETING_DIR'])}/{channel}"
         for datetxt in datelist:
-            datetxt_dict[datetxt] = "https://meetbot-raw.fedoraproject.org/%s/%s" % (
-                channel,
-                datetxt,
-            )
+            datetxt_dict[datetxt] = f"{app.config['MEETBOT_URL']}/{channel, datetxt}"
```

Comment:
Make sure the original formatting is respected.
It should be:
```suggestion
            datetxt_dict[datetxt] = f"{app.config['MEETBOT_URL']}/{channel}/{datetxt}"
```